overkill_nvim/src/lib.rs

```diff
@@ -21,9 +21,12 @@
 )]
 
 pub mod key_code;
+pub mod mapping;
+// pub mod mode;
```

Comment:
nuke

Reply:
Actually, keep. I left the module file in the repo so I can readd stuff in the future.